packaging/amazon-linux-ami/ecs-init.spec

```diff
@@ -399,16 +399,16 @@ fi
 * Tue Nov 21 2017 Noah Meyerhans <nmeyerha@amazon.com> - 1.16.0-1
 - Cache Agent version 1.16.0
 
-* Tue Nov 14 2017 Noah Meyerhans <nmeyerha@amazon.com> - 1.15.2-1
+* Wed Nov 15 2017 Noah Meyerhans <nmeyerha@amazon.com> - 1.15.2-1
 - Cache Agent version 1.15.2
 
 * Tue Oct 17 2017 Jacob Vallejo <jakeev@amazon.com> - 1.15.1-1
 - Update ECS Agent version
 
-* Fri Oct 06 2017 Justin Haynes <jushay@amazon.com> - 1.15.0-1
+* Sat Oct 07 2017 Justin Haynes <jushay@amazon.com> - 1.15.0-1
 - Update ECS Agent version
 
-* Fri Sep 29 2017 Justin Haynes <jushay@amazon.com> - 1.14.5-1
+* Sat Sep 30 2017 Justin Haynes <jushay@amazon.com> - 1.14.5-1
 - Update ECS Agent version
 
 * Tue Aug 22 2017 Justin Haynes <jushay@amazon.com> - 1.14.4-1
```

Comment:
any reason these below ones remain unchanged?

Reply:
These are generated from a source timestamp in https://github.com/aws/amazon-ecs-init/blob/master/scripts/changelog/CHANGELOG_MASTER

Because this spec file doesn't have the timezone, the date would only change if the source timestamp was after 4pm PST, because anything after 4pm PST would roll-over into the next day in UTC.